packages/app-info/lib/index.js

```diff
@@ -125,7 +125,20 @@ module.exports = {
 	 * @readonly
 	 * @type {string | null}
 	 */
-	processType
+	processType,
+
+	/**
+	 * The cloud provider type.
+	 *
+	 * @readonly
+	 * @type {string | null}
+	 */
+	cloudProvider:
+		(processType === process.env.AWS_LAMBDA_FUNCTION_NAME
+			? 'aws'
+			: processType === process.env.DYNO
+				? 'heroku'
```

Comment:
eslint is not passing on line 140 and 141 due to it expecting 4 tabs but got 3, but when I change it to 4 tabs then prettier complains 🥲 . Got around this by passing `--no-verify` in the terminal but will need to fix for CircleCI to pass I'd imagine

Reply:
Changed code completely